app/src/main/kotlin/com/wire/android/ui/authentication/create/details/CreateAccountDetailsScreen.kt

```diff
@@ -127,174 +125,179 @@ private fun DetailsContent(
+                                .padding(
+                                    start = MaterialTheme.wireDimensions.spacing16x,
+                                    end = MaterialTheme.wireDimensions.spacing16x,
+                                    bottom = MaterialTheme.wireDimensions.spacing16x
+                                )
+                                .testTag("teamName")
+                                .focusRequester(focusRequesterTeamName),
+                            shouldDetectTaps = true,
+                            onTap = {
+                                coroutineScope.launch {
+                                    keyboardController?.show()
+                                    focusRequesterTeamName.requestFocus()
+                                    listState.animateScrollToItem(2)
+                                }
+                            }
                         )
-                )
-                NameTextFields(state, onFirstNameChange, onLastNameChange, onTeamNameChange, coroutineScope)
-                PasswordTextFields(state, onPasswordChange, onConfirmPasswordChange, coroutineScope)
-                Spacer(modifier = Modifier.weight(1f))
-            }
+                    }
+                }
 
-            Surface(
-                shadowElevation = scrollState.rememberBottomBarElevationState().value,
-                color = MaterialTheme.wireColorScheme.background
-            ) {
-                Box(modifier = Modifier.padding(MaterialTheme.wireDimensions.spacing16x)) {
-                    WirePrimaryButton(
-                        text = stringResource(R.string.label_continue),
-                        onClick = onContinuePressed,
-                        fillMaxWidth = true,
-                        loading = state.loading,
-                        state = if (state.continueEnabled) WireButtonState.Default else WireButtonState.Disabled,
-                        modifier = Modifier.fillMaxWidth()
+                item {
+                    WirePasswordTextField(
+                        value = state.password,
+                        onValueChange = onPasswordChange,
+                        labelMandatoryIcon = true,
+                        descriptionText = stringResource(R.string.create_account_details_password_description),
+                        imeAction = ImeAction.Next,
+                        autofillTypes = listOf(),
+                        modifier = Modifier
+                            .padding(horizontal = MaterialTheme.wireDimensions.spacing16x)
+                            .testTag("password"),
+                        state = if (state.error is CreateAccountDetailsViewState.DetailsError.TextFieldError.InvalidPasswordError) {
+                            WireTextFieldState.Error()
+                        } else {
+                            WireTextFieldState.Default
+                        }
+                    )
+                }
```

Comment:
Same with these two password items - what if someone has shorter screen and after scrolling to `teamName`, both password inputs still won't be fully visible? Shouldn't all inputs have `onTap` with scrolling implemented? 🤔

Reply:
Even with shorter screen, user should see password field. It's just bellow `teamName`.
I didn't apply for all fields as it could be a bad UX.

Reply:
https://github.com/wireapp/wire-android-reloaded/assets/30429749/11f3cb43-feb2-4b04-86b8-9c9533e47800

This is what I tried to explain, so it doesn't always scroll and even if the user clicks on "password", then "confirm password" is still hidden under the "continue" button

Reply:
Weird! The app is not scrolling at all to the item . What's this device ?

Reply:
Pixel 7 with Android 13. 
It's scrolling properly when I go in order from "first name" to "last name" and then to "team name", but it doesn't if I open "last name" and then click on "team name" or if I open "first name" and go directly to "team name". Tapping on "team name" only scrolls if the whole list is already scrolled at least by a pixel or if "team name" is already focused and I tap on that again.

Reply:
I suspect that if the input is not fully on screen, even slightly by 1px, then scroll tries automatically to bring it into screen, because `LazyColumn` has `keepFocusedChildInViewModifier` inside, which is `bringIntoViewResponder`, so if the input is even slightly off screen then this `bringIntoView` interferes with our `animateScrollToItem` and makes it so it actually doesn't scroll. When I added a delay right before `animateScrollToItem` then it started working because our scroll was happening after the `bringIntoView` is completed.

Reply:
I would say let's keep it simple and not waste time on it, the user is here for registration not for detecting edge cases UI bugs 🐛  😛
If the user skips some fields then he must come back to fill them and scrolling will work. ✅
If he do, like you did in video by hiding keyboard every time, then he must notice about remaining fields ✅

Please remember the aim of this PR is to make the user aware about other fields and not get blocked.

Reply:
Personally I would prefer to show scroll bar instead of this idea, but it's not possible yet with LazyColumn